mms/benchmarks/benchmark.py

```diff
@@ -184,12 +184,18 @@ def run_single_benchmark(jmx, jmeter_args=dict(), threads=100, out_dir=None):
             else:
                 docker_run_call = "{} run --name {} -p 8080:8080 -v {}:/mxnet-model-server -itd {}".format(docker, container, MMS_BASE, docker_path)
                 run_process(docker_run_call)
-                run_process("{} exec -it {} sh -c 'cd /mxnet-model-server && pip install -U -e .'".format(docker, container), shell=True)
-
-
-        with ChDir(MMS_BASE):
-            run_process('python setup.py bdist_wheel --universal')
```

Comment:
Did you delete this line?

Reply:
Yes. This was being done after pip install -U -e .

It has to be a precursor to it.